runtime.md

```diff
@@ -22,6 +22,8 @@ This allows the hooks to perform cleanup and teardown logic after the runtime de
 * **`pid`**: (int) is the ID of the main process within the container, as seen by the host.
 * **`bundlePath`**: (string) is the absolute path to the container's bundle directory.
 This is provided so that consumers can find the container's configuration and root filesystem on the host.
+* **`Labels`**: (map) Labels is the runtime itself specified information.
+This allows runtime to store some informations which are usefull for itself but meaningless for other runtime.
```

Comment:
I'd keep the first line of this the same as the `config.go` comment (whatever that ends up being).  For the second line, maybe “The runtime could store experimental information here before it's standardized in this specification (e.g. the socket path used to control runV virtual machines)”.